config/default.js

```diff
@@ -1,20 +1,21 @@
-const _ = require('lodash');
-
 module.exports = {
   bugsnag: {
-    token: _.get(process.env, 'BUGSNAG_TOKEN', null),
+    token: process.env.BUGSNAG_TOKEN || null,
   },
   database: {
     connectionString: process.env.CONNECTION_STRING,
   },
-  eventType: 'LogFill',
-  genesisBlock: 4145578,
-  maxChunkSize: 10000,
+  maxChunkSize: parseInt(process.env.MAX_CHUNK_SIZE, 10),
   minConfirmations: 12,
-  pollingInterval: 30000,
-  protocolVersion: 1,
+  pollingInterval: parseInt(process.env.POLLING_INTERVAL, 10),
+  v1: {
+    genesisBlock: 4145578,
+  },
+  v2: {
+    genesisBlock: 8140780,
```

Comment:
This actually maps to 0x v2.1. Work will be done to distinguish between v2 and v2.1 in the future.